instrumentation/othttp/handler.go

```diff
@@ -99,6 +124,8 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	ctx, span := h.tracer.Start(ctx, h.spanNameFormatter(h.operation, r), opts...)
 	defer span.End()
 
+	requestStartTime := time.Now()
```

Comment:
Should this move to the top of the method to capture the startup overhead?

Reply:
+

Reply:
done